package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@inventive/sdk",
   "description": "SDK library for types, function calls and React components to embed Inventive content in any web app",
   "author": "Inventive X",
-  "version": "0.0.1",
+  "version": "0.0.1.1",
```

Comment:
@kateheesunlee no, semantic versioning only has 3 parts
https://docs.npmjs.com/about-semantic-versioning
major.minor.patch

I was suggesting that you make this 0.1.0, not add a 4th (non-existent) category of versioning

Reply:
😅 yes I noticed that from Pin-yen's PR.